packages/postcss-svgo/src/__tests__/index.js

```diff
@@ -203,19 +209,6 @@ test(
   )
 );
 
```

Comment:
I have changed these tests because I 've realized there is no meaningful difference between these cases. They're all invalid SVGs. They were handled separately because some failed at the `isSvg` stage and some at the `svgo` stage.